lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,10 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, options = {})
```

Comment:
Is het ook leuk om named arguments te gebruiken voor een stukje validatie?
`sbg_domain(sbg_key, begin:, end: , outcome:, key: sbg_key, primary:false)`